.github/workflows/actions/add-spack-googletest/action.yml

```diff
@@ -0,0 +1,23 @@
+name: "Add Googletest via Spack"
+description: |
+  Checks to see if Googletest is installed in a Spack environment
+  and adds/installs it if not. Then adds an environment module
+  and adds a load of said module to the .bashrc.
+
+runs:
+  using: "composite"
+  steps:
+    - name: "Add Googletest"
+      shell: bash -l {0}
+      run: |
+          spack find -I googletest
+          have_gtest=$?
+          if [ -z "$have_gtest" ]
+          then
+            echo "REMARK: Googletest already installed; can remove this temporary step now"
+          else
+            spack add googletest
+            spack install googletest
+            spack module tcl refresh -y googletest
+            echo -e "\nmodule load googletest" >> /home/runner/.bashrc
+          fi
```

Comment:
> Also requires adding Googletest to the environment, which I added via a small action. Long term approach will be to add this to all of the container images.

@sebrowne, what is the timetable for doing this?  For our usage of Trilinos containers in our project, we can just update our fork's Dockerfiles and build our own and add googletest to the main spack build.  But for other projects and other developers, this will make it much harder for developers to reproduce Trilinos builds using the containers.  They will have to manually run the above spack commands every time they start a container?

Is it not easy to update the Dockerfile files for the containers and run 'docker build' and post the new images?  If this is the only thing that changes, that should be pretty safe, no?